pkg/processresetqueue/reset_test.go

```diff
@@ -470,6 +470,23 @@ func TestReset(t *testing.T) {
 	})
 }
 
+func TestExtractRoleNameFromARN(t *testing.T) {
```

Comment:
I made a minor change to this method, to fix a linting error. 
Just wanted to make sure I didn't break anything